### ISSUE TITLE
Transition animation doesn't reset after closing/reopening

### ISSUE THREAD

**Issue by kevinkucharczyk** [cibernox/ember-basic-dropdown#615]
This can be seen even in the docs: https://ember-basic-dropdown.com/docs/animations/

The `transitioning--in` etc classes don't get reset after the first render. When the dropdown content first renders, it animates correctly, but on subsequent renders there is no animation - it appears instantly.

Here'a a screencast:
![2021-05-20 11-27-10 2021-05-20 11_27_30](https://user-images.githubusercontent.com/3127419/118905120-7b33f880-b95e-11eb-9b73-88267164ee79.gif)

You can see how on the first click in smoothly animates in, but after that it doesn't.

There also doesn't seem to be a transitioning-out animation, even though it's defined in the CSS.

**Comment by denzo** [comment]
Yes, I am experiencing the same in `ember-power-select` as it uses this addon under the hood.

Can't find `dropdown--transitioning-out` in the codebase at all also.

**Comment by mkszepp** [comment]
closing this old issue... looks like it was fixed already (at least in our docs it works)
